### ISSUE TITLE
Update default user agent and change it to follow Chromium user agent reduction

### ISSUE THREAD

**Issue by fourquestionmarksokay** [tachiyomiorg/tachiyomi-extensions#12472]
### Describe why this should be added

Issues #12467, #12449, and #12465 relates to the fact that an old UA (`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36 Edg/88.0.705.63`) being currently used is blacklisted by CF, even if you changed the user agent of currently-installed Edge to the older one (although a) it seems that ALL valid user agents before Edge 98 is blocked by CF, and b) it seems that if you happened to be on an actual Edge 88 CF will actually properly redirect, you can verify this by downloading and installing the old version from https://web.archive.org/web/20210212222010/https://www.microsoft.com/en-us/edge/business/download and executing `msiexec /i MicrosoftEdgeEnterpriseX64.msi ALLOWDOWNGRADE=1`, which I am surprised that this method works in the first place). 

At the same time Chromium is currently doing an [update to reduce their user agent into the major version and some idea of the platform](https://www.chromium.org/updates/ua-reduction/#sample-ua-strings-phase-6), which we can take advantage by updating the user agent following the [current canary channel](https://en.wikipedia.org/wiki/Google_Chrome_version_history), which is currently version 105. The reason we're tracking Canary is that it's more likely that user agent reduction is enabled on dev builds. The current stable version 103 only minimizes the version number and not the specific device identification and Android version (corresponding to [Phase 4](https://www.chromium.org/updates/ua-reduction/#sample-ua-strings-phase-4) of the user agent reduction).

Combining that, it will produce the user agent would be currently `Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Mobile Safari/537.36`, regardless of their actual device or Android version. It will require tracking Canary until at least version 113 (where user agent reduction is expected to be implemented in the stable version). I have only tested this in Kiwi Browser (which is the closest Chromium browser with devtools in Android), but I haven't verified it yet on an actual WebView implementation or extension.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
